config/webpack.config.js

```diff
@@ -43,6 +43,10 @@ const configGenerator = (buildOptions, apps) => {
     ENVIRONMENTS.VAGOVPROD,
   ].includes(buildOptions.buildtype);
 
+  const enableCSSSourcemaps =
+    buildOptions.buildtype !== ENVIRONMENTS.LOCALHOST ||
+    buildOptions['local-css-sourcemaps'];
```

Comment:
If `buildOptions['entry']` is passed, should it flip to `true`? Just thinking that if we've limited what apps we're compiling, then maybe it should consider CSS sourcemaps to be safe too.